.sbtopts

```diff
@@ -1 +1,2 @@
 -Dfile.encoding=UTF-8
+-J-Xmx1G
```

Comment:
Why the change here? It shouldn't need `-J` anyway

Reply:
`.sbtops` needs `-J` prefix, but I'll remove this option anyway - newer JDKs don't have 512M limit as jdk8 has